sdk/ml/azure-ai-ml/azure/ai/ml/operations/_model_operations.py

```diff
@@ -142,13 +143,16 @@ def create_or_update(
                             error_category=ErrorCategory.USER_ERROR,
                         )
 
-                    model = model._to_rest_object()
+                    model_rest_obj = model._to_rest_object()
```

Comment:
Out of consistency, the other file named the variable `environment_rest`
Not picky about which naming we use